package.json

```diff
@@ -23,15 +23,17 @@
     "build:server": "NODE_ENV=production webpack --config ./webpack/server.babel.js",
     "build:db": "node ./src/server/database/setupDB.babel.js",
     "update:db": "node ./src/server/database/setupDB.babel.js true",
-    "test": "NODE_ENV=testing ava ./src/**/__tests___/**/*.js --verbose",
```

Comment:
my personal preference is to keep these separate. That way, when we write & debug tests they run screamingly fast. The CircleCI config will run both test & lint anyways, so no worries about accepting bogus PRs.